### PR TITLE
Bump Ubuntu and Mapserver. 

### PR DESCRIPTION
**Pull request by emacgillavry**
Operating system updated to 22.04. MapServer minor version update picks up correct projection params from 'epsg' file.

### CODE REVIEW

**Review by lennardstam — APPROVED** [review]
OK

**Comment by lbam** [comment]
`Dockerfile.private` needs the same changes, right? Do we do that now, or in a separate PR?

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+        gdal-bin \
+        gdal-data \
+        mapserver-bin \
+        python3-pip \
+        wget \
+    && apt-get clean
 
-RUN apt-get install -y gdal-bin gdal-data libgdal20
-RUN apt-get install -y apache2 apache2-utils libmapcache1 libapache2-mod-mapcache cgi-mapserver mapserver-bin
+RUN python3 -m pip install mappyfile==0.9.7
```
[the remark this answers is not in the document]
Comment (lbam) [reply Dockerfile:18]:
```suggestion
```

[the remark this answers is not in the document]
Comment (emacgillavry) [reply Dockerfile]: Top idee!

**Comment by emacgillavry** [comment]
De `Dockerfile.private` is een multi-stage build, obv de standaard Dockerfile, toch? Dus die neemt de aanpassingen klakkeloos over lijkt me

**Pull request merged**